ci/cscs.yml

```diff
@@ -0,0 +1,26 @@
+include:
+  - remote: 'https://gitlab.com/cscs-ci/recipes/-/raw/master/templates/v2/.ci-ext.yml'
+
+stages:
+  - configure
+  - run
+
+pipeline-configure:
+  stage: configure
+  extends: .container-runner-lightweight-zen2
+  image: docker.io/rjanalik/ci-generator:python-3.12.3
```

Comment:
It's fine to use this image, but could you add the Dockerfile to this repository, so I could be understood what is inside?